MonoGame.Framework/Platform/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -143,38 +138,57 @@ internal void ResolveSubresource()
             lock (GraphicsDevice._d3dContext)
             {
                 GraphicsDevice._d3dContext.ResolveSubresource(
-                    _msTexture,
+                    GetMSTexture(),
                     0,
-                    _texture,
+                    GetTexture(),
                     0,
                     SharpDXHelper.ToFormat(_format));
             }
         }
 
-        internal override void CreateTexture()
+        protected internal override Texture2DDescription GetTexture2DDescription()
```

Comment:
instead of CreateTexture() we overide GetTexture2DDescription() to modify the Texture2DDescription.
creation of _msTexture  is moved to a new method, CreateMSTexture().